packages/common/src/types/UnixTime.ts

```diff
@@ -105,4 +105,8 @@ export class UnixTime {
   toJSON() {
     return this.timestamp
   }
+
+  toDateDays() {
```

Comment:
Maybe `toYYYYMMDD()` or `toFormat('YYYY-MM-DD')` where `"YYYY-MM-DD"` would be the only accepted parameter.